src/components/admin/ExpenseSummary.vue

```diff
@@ -201,6 +207,10 @@ export default defineComponent({
     }
   },
   methods:{
+    printDiv(div_id: string, title: string) {
```

Comment:
you cannot call printDiv directly ?

Reply:
No we have to declare all imported functions or components in script.